packages-next/admin-ui/src/pages/ListPage/FilterAdd.tsx

```diff
@@ -125,22 +125,32 @@ function FilterAddPopoverContent({ onClose, listKey }: { onClose: () => void; li
   }, [router.query, fieldsWithFilters]);
   const [state, setState] = useState<State>({ kind: 'selecting-field' });
 
+  const updateFilters = () => {
+    if (state.kind === 'filter-value') {
+      router.push({
+        query: {
+          ...router.query,
+          [`!${state.fieldPath}_${state.filterType}`]: JSON.stringify(state.filterValue),
+        },
+      });
+      onClose();
+    }
+  };
+
   return (
     <Stack
       padding="medium"
       as="form"
+      onKeyDown={(event: React.KeyboardEvent<HTMLFormElement>) => {
+        if (event.code === 'Enter') {
```

Comment:
Why does an enter not trigger the submit event?

Reply:
Ah... I thought it was react-select hijinx, but turns out its #justbuttonthings. 
Button didn't have an explicit type, so its hijacking form events in the onClick (which is also triggered by an enter key press) 

Fixed and pushed